client/galaxy/scripts/store/configStore/queries.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+import { prependPath } from "utils/redirect";
+
+export async function getConfig() {
+    const url = prependPath("/api/configuration");
```

Comment:
I suspect this code is part of the javascript run on toolshed, which does not have this endpoint, hence broken -https://github.com/galaxyproject/galaxy/issues/9061